src/liger_kernel/transformers/monkey_patch.py

```diff
@@ -151,12 +152,15 @@ def apply_liger_kernel_to_mllama(
         MllamaForCausalLM,
         MllamaForConditionalGeneration,
         MllamaTextModel,
+        MllamaVisionModel,
     )
 
     from liger_kernel.transformers.model.mllama import lce_forward as mllama_lce_forward
 
     if rope:
         modeling_mllama.apply_rotary_pos_emb = liger_rotary_pos_emb
+    if layer_norm:
+        modeling_mllama.nn.LayerNorm = LigerLayerNorm
```

Comment:
FWIW I looked into this but was worried this would modify torch.nn.LayerNorm globally, not just for the targeted model

Reply:
That probably answers why ci tests fail. LigerLayerNorm gets applied even before we monkey patch because a previous test patched layer norm globally.